spring-web-modules/spring-web-url/src/main/java/com/baeldung/exclude_urls_filter/Application.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.exclude_urls_filter;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@ComponentScan(basePackages = "com.baeldung.exclude_urls_filter")
```

Comment:
We don't need it since we already use @SpringBootApplication annotation

Reply:
Fixed it